staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation_test.go

```diff
@@ -2040,7 +2040,7 @@ func TestValidateCustomResourceDefinition(t *testing.T) {
 										},
 										"bad": {
 											Type:    "string",
-											Pattern: "+",
```

Comment:
make a new test with just this bit so we don't lose coverage of schemas with invalid regexes?

Reply:
we have that elsewhere https://github.com/kubernetes/kubernetes/pull/78453/files#diff-3a143542772c2362df0e74a0c908ff76R239